at_secondary/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -249,6 +258,21 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
     }
   }
 
+  AtMetaData? _removeKeyFromMetadataCache(String key) {
+    // try removing the passed key as it is. key may be encoded or not encoded.
```

Comment:
Cache is created upon process start, so shouldn't we know definitively how to remove the key from the cache?

Reply:
you are right Gary.. After the fix when we restart the server, cache should have keys in correct format.I will re-test this scenario and fix the code.